source/client-side-encryption/client-side-encryption.rst

```diff
@@ -838,8 +844,27 @@ Identifies a key vault collection document by 'keyAltName'.
 
 algorithm
 ^^^^^^^^^
-The string "AEAD_AES_256_CBC_HMAC_SHA_512-Deterministic" or
-"AEAD_AES_256_CBC_HMAC_SHA_512-Random"
+One of the strings:
+- "AEAD_AES_256_CBC_HMAC_SHA_512-Deterministic"
+- "AEAD_AES_256_CBC_HMAC_SHA_512-Random"
+- "IndexedEquality"
+- "Unindexed"
+
+The result of explicit encryption with the "IndexedEquality" algorithm must be processed by the server to insert or query. Drivers MUST document the following behavior:
+
+   To insert or query with an "IndexedEquality" encrypted payload, use a ``MongoClient`` configured with ``AutoEncryptionOpts``.
+   ``AutoEncryptionOpts.bypassQueryAnalysis`` may be true. ``AutoEncryptionOpts.bypassAutoEncryption`` must be false.
+
+contentionFactor
+^^^^^^^^^^^^^^^^
+contentionFactor only applies when algorithm is "IndexedEquality".
+It is an error to set contentionFactor when algorithm is not "IndexedEquality".
+Setting a higher contentionFactor adds randomization. It is useful when values being encrypted have low cardinality.
```

Comment:
Can we be more precise than "higher"?  I would have no idea what to set this to given this documentation.  Also, what's the default value if it's unset?  Is it an error for the value to be <= 0?

Reply:
> Can we be more precise than "higher"? I would have no idea what to set this to given this documentation.

I removed this sentence. I do not know what to recommend for contentionFactor.

> Also, what's the default value if it's unset?

The default is 0. I have added a sentence to clarify.

> Is it an error for the value to be <= 0?

Good question. It is not in libmongocrypt currently. The server will error if attempting to create a collection with an negative contentionFactor listed in encryptedFields: `(Location51024) BSON field 'contention' value must be >= 0, actual value '-1'`. Filed [MONGOCRYPT-425](https://jira.mongodb.org/browse/MONGOCRYPT-425) to make the change in libmongocrypt.